frontend/projects/upgrade/src/app/features/dashboard/experiment-users/components/experiment-users/experiment-users.component.ts

```diff
@@ -69,15 +68,19 @@ export class ExperimentUsersComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit() {
+    this.experimentService.fetchContextMetaData();
     this.permissions$ = this.authService.userPermissions$;
     this.allExcludedEntities.paginator = this.paginator;
     this.allExcludedEntities.sort = this.sort;
 
     this.contextMetaDataSub = this.experimentService.contextMetaData$.subscribe(contextMetaData => {
-      this.contextMetaData = contextMetaData;
-      if (this.contextMetaData && this.contextMetaData['groupTypes']) {
-        this.contextMetaData['groupTypes'].forEach(element => {
-          this.groupTypes.push({value: element});
+      this.contextMetaData = contextMetaData; 
+      if( contextMetaData ) {
```

Comment:
@RidhamShah Can you please fix the spacing issue here inside the if condition?

Reply:
code updated